### ISSUE TITLE
0.10.4 has a List.sum error

### ISSUE THREAD

**Issue by christinachimi** [madewithfelt/flutter_layout_grid#36]
We're using a library that depends on flutter_layout_grid, and our builds started to fail this evening with this error:

```
../../_tool/flutter/1.22.5-stable/x64/.pub-cache/hosted/pub.dartlang.org/flutter_layout_grid-0.10.4/lib/src/rendering/layout_grid.dart:1003:35: Error: The getter 'sum' isn't defined for the class 'List<double>'.
 - 'List' is from 'dart:core'.


Try correcting the name to the name of an existing getter, or defining a getter or field named 'sum'.
    return baseSizesForType(type).sum;

                                  ^^^
```

I think we can pin an override but wanted to report.

**Comment by shyndman** [comment]
Hey!

Pin your build at 0.10.3, and follow this issue. I'll have it fixed up for you tomorrow.

Sorry about that.

**Comment by shyndman** [comment]
Does your project have extension methods enabled?

Would you mind copying your `flutter --version` to this issue? And your `analysis_options.yaml`?

**Comment by shyndman** [comment]
You know what, I think I know what's wrong. Give me a few minutes and I'll put up a new patch.

**Comment by shyndman** [comment]
OK, version `0.10.5` is up. I think your build should work now.

And sorry about this. This package uses semantic versioning, but this new Dart null-safety stuff put me in a position where I had to choose between `pub` complaining and semver, and I didn't think I'd break anybody.

**Comment by christinachimi** [comment]
Looks good, thank you for the quick fix!

**Comment by shyndman** [comment]
Fixed by #67ac10c
